packages/modal/src/presenters/ModalHeaderPresenter.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import IconButton from "@hig/icon-button";
+
+import "./ModalHeaderPresenter.scss";
+
+export default class ModalHeaderPresenter extends Component {
+  static propTypes = {
+    /**
+     * Supports adding any dom content to the header of the modal
+     */
+    children: PropTypes.node,
+    /**
+     * Triggers when one clicks the close button
+     */
+    onCloseClick: PropTypes.func,
+    /**
+     * Title of the modal
+     */
+    title: PropTypes.string
+  };
+
+  renderChildren() {
+    return (
+      <header className="hig__modal-V1__header">{this.props.children}</header>
+    );
+  }
+
+  render() {
+    return this.props.children ? (
+      this.renderChildren()
+    ) : (
+      <header className="hig__modal-V1__header">
+        <IconButton
+          aria-label="close"
+          name="x-close-gray"
+          onClick={event => this.props.onCloseClick(event)}
+          title="Close"
```

Comment:
Let add this value via a prop to allow for i18n. Use the same one for `aria-label`.